### ISSUE TITLE
Duplication not working for JSON fields

### ISSUE THREAD

**Issue by formfcw** [formfcw/directus-extension-flexible-editor#13]
**Describe the bug**
Duplication not working for related collections that have JSON fields, as long as they are selected in the item duplication fields.

This should be solved with the planned rewrite: https://github.com/formfcw/directus-extension-flexible-editor/discussions/11

**Flexible Editor Version:**
1.0.0

**Directus Version:**
10.4.3

**Comment by alphanull** [comment]
Are you still planning to implement this? Otherwise I might open a PR for it.

**Comment by formfcw** [comment]
Thank you @alphanull :heart: That would be great! Please ensure to submit a reviewable PR — I can’t accept AI slop. We don’t have much time to review PRs at the moment, so please keep it precise, clean and in scope. Much appreciated 🙏🏻

**Comment by alphanull** [comment]
@formfcw, well if I contribute, I definetly will be using AI, otherwise not possible. Is that OK for you? Other than that I also have other ideas how to improve this extension. Actually, I think this belongs into directus core! I mean still dealing with HTML in a CMS is soooo 90s ;)

So for me the questions would be: what is the planned future of this extension and is it OK using AI? (and another question is if I actually want to stay with directus, with the latest developments regarding licensing deeply concern me, but thats another topic)

**Comment by formfcw** [comment]
Hey, @alphanull 👋🏻 An AI assisted workflow is no problem, of course. As long as you use best practices and self-review before submitting your PR it’s totally fine. I agree with the 90s part … haha 😆 

> (and another question is if I actually want to stay with directus, with the latest developments regarding licensing deeply concern me, but thats another topic)

I understand your concerns. This change may look frightening at first. However, the recent changes primarily just solidify the terms that were previously present: so long as you're under $5M revenue and 50 employees, you're still eligible for an OIG license which takes less than 1 minute to [apply for & receive](https://directus.com/oig). Highly recommend watching [this short video](https://directus.com/resources/directus-v12-license-change). If you’d like to discuss this further, please reach out on the Directus Discord channel, as it is not related to this issue. ❤️

**Comment by alphanull** [comment]
Actually this change IS frightening and basically threatening my whole business model. You know there are quite a LOT of companies over 5M revenue but with almost no earnings, especially here in Germany. I have no ideas in which outer spheres you guys reside, but this is the world _I_ see. Tell those companies to shell out 1000$ a month for a CMS license (not even including a cloud or anything) and they will laugh at you. Very hard. And I understand, why should they. For 1000$ a month there are more solid offers anyway. And then one day, I add another collection to my system, and the next day I ask my customer to give me $100 more. Yeah, sure. Oh wait I already have 54 collections. And I still have no idea what "Enterprise" actually costs. No information other than "ask us". Besides that, only rumors, and no answer to my inquiry so far. Also DRM like enforcements, unclear data transmission etc. AI tranlsations -  enterprise feature? Seriously?????

Sorry, this is a total mess, and I am the more shocked, the more I find out about it. Basically you want to serve the enterprise, allow free use for some basic stuff, and give a sH*t about the rest. Sorry to put it so bluntly but thats the way I see it right now.

**Comment by alphanull** [comment]
And of course AI work will be assisted, plan & spec driven and not just some random vibe coding. And I really would like to contribute, as I said your project should be actually a first class citizen. I centered everything around it despite the added complexity. The only way to use my own blocks in a sensible way, and the only way to have truly structured content, no HTML soup. So, good work, I like it!

I also have done some other directus extensions on my own, and actually I wanted to release them to the public (for instance I have a rely nice directus backup solution on my plate - art least imho) .... but since the newest announcements, I am not sure if it is actually a good idea to out any more energy into directus. So until I get some definite answers, I will put all development on hold.

**Comment by formfcw** [comment]
@alphanull Let’s discuss this on [Discord](https://discord.com/invite/directus) 🙏🏻 This repo and specifically this issue is not the right place for it. Hence, I will mark your response as off-topic. However, the Directus team sees your pain, and we are happy to discuss this in our community channels. Discord is the place where we can help you and find a solution.

**Comment by alphanull** [comment]
You are right, sorry for the rant. I am just totally floored and confused. Sorry.
